controllers/k8score/event_controller.go

```diff
@@ -76,7 +76,7 @@ var _ EventHandler = &RoleChangeEventHandler{}
 
 func init() {
 	EventHandlerMap["role-change-handler"] = &RoleChangeEventHandler{}
-	EventHandlerMap["consensus-set-event-handler"] = &consensusset.PodRoleEventHandler{}
```

Comment:
fix up UT coverage for this package

Reply:
done, coverage to 80%: [edc2a4a](https://github.com/apecloud/kubeblocks/pull/3733/commits/edc2a4a2e8d56bbf4af3f9425b905a06f60342ec)